README.md

```diff
@@ -31,6 +31,7 @@
   - Crop
   - Gaussian blur
   - Horizontal blur
+  - Padding
```

Comment:
I think it's a bit confusing that this differs from the method name, but I agree that it might be a bit awkward to have "Pad". Do you have any ideas how we might handle this?

Reply:
Maybe by renaming the module to `Padding` and the methods to `#padding` and `#padding!`?

Reply:
`Padding` it is! Works for me :+1: